src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducerName: 'reducerFunction',
+});
+
+export default store;
```

Comment:
- [ ] Well done working on setting up the store 👏. However, to complete the configuration, I would recommend you make sure that the `booksSlice` and the `categoriesSlice` are imported and made available in the `configureStore`. 👌 😊

```suggestion
import { configureStore } from '@reduxjs/toolkit';
import booksReducer from './books/booksSlice';
import categoriesReducer from './categories/categoriesSlice';

const store = configureStore({
  books: booksReducer,
  categories: categoriesReducer,
});

export default store;
```